core/ledger/kvledger/txmgmt/statedb/statecouchdb/couchdb_test_export.go

```diff
@@ -22,6 +22,16 @@ func StartCouchDB(t *testing.T, binds []string) (addr string, stopCouchDBFunc fu
 	return couchDB.Address(), func() { couchDB.Stop() }
 }
 
+// StartCouchDB starts the CouchDB if it is not running already
+func StartLegacyCouchDB(t *testing.T, binds []string) (addr string, stopCouchDBFunc func()) {
```

Comment:
As it is present in the `couchdb_test_export.go`, I have to assume that the test would be run only using CouchDB 2.3. I don't know what problem we would face/introduce due to not testing with both CouchDB 2.3 and 3.1.

Reply:
The proposal here is all testing is done using 3.1, the only testing done on 2.3 is the auto-creation of the `_users` table. Unless we think it should be reversed, all testing on 2.3, with only the _users case tested on 3.1. But my opinion is that we should move forward and 3.1 should become our new standard.